spark_log_parser/parsing_models/application_model_v2.py

```diff
@@ -17,34 +17,36 @@
 class sparkApplication:
     def __init__(
         self,
-        objfile=None,  # Previously saved object. This is the fastest and best option
-        appobj=None,  # application_model object
-        eventlog=None,  # spark eventlog path,
+        spark_eventlog_parsed_path=None,
+        spark_eventlog_path=None,  # application_model object
+        appobj=None,
         stdout=None,
         debug=False,
     ):
 
-        self.eventlog = eventlog
+        self.eventlog = spark_eventlog_path
         self.existsSQL = False
         self.existsExecutors = False
         # self.sparkMetadata = {}
         self.metadata = {}
         self.stdout = stdout
         self.debug = debug
 
-        if objfile is not None:  # Load a previously saved sparkApplication Model
-            self.load(filepath=objfile)
+        if spark_eventlog_parsed_path is not None:  # Load a previously saved sparkApplication Model
+            self.load(filepath=spark_eventlog_parsed_path)
 
-        if (appobj is not None) or (eventlog is not None):  # Load an application_model or eventlog
+        if (appobj is not None) or (
+            self.eventlog is not None
```

Comment:
Wasn't this attribute renamed?

Reply:
No `objfile` was renamed. `appobj` is a very old option that doesn't get used anymore.

Reply:
on second thought, I'll clean this up a bit more to remove appobj and make sure attributes are all consistent.